package.json

```diff
@@ -29,31 +27,44 @@
     "Google Cloud Key Management Service (KMS) API"
   ],
   "scripts": {
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test": "c8 mocha",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "prelint": "gts fix && cd samples/ && npm install && cd .. && eslint --fix samples/*.js",
```

Comment:
This doesn't look right.  I think it's supposed to be:

```
 "prelint": "cd samples; npm link ../; npm i",
```

Reply:
re-setup the scripts and tested it, thanks!